README.md

```diff
@@ -172,6 +172,71 @@ lein build-prod
 node server/name-bazaar.js
 ```
 
+## Testnet deploy
+
+To run server in docker container use image `district0x/namebazaar-server`, e.g.:
+
+```bash
+docker run --name=namebazaar-server \
+    --net=host \
+    -v /path/to/config.edn:/configs/namebazaar.config.edn \
+    district0x/namebazaar-server:latest
+```
+
+You can choose between tags `dev`, `latest` and `release`.  As for the config file, you can find an example in `docker-builds/server/config.example.edn`. Of particular interest is providing correct addresses of smart contracts on the blockchain you'll link the app to.
+
+For UI use the `district0x/namebazaar-ui` image:
+
+```bash
+docker run --name=namebazaar-ui \
+    --net=host \
+    district0x/namebazaar-ui:latest
+```
+
+Note that there is no passing of config file for UI: currently for any change of UI config you need to build a new image (see the next section). The hardcoded configuration is at `src/name_bazaar/ui/config.cljs`.
+
+### Updating docker images
+
+If you want to build new docker images locally and push them to district0x dockerhub (if you're authorised to do so), run:
+
+```bash
+./docker-push.sh env sshkey
+```
+
+where
+
+* `env` is `dev`,`qa` or `prod` (the only difference is in how the images will be tagged)
+* `sshkey` is path to your private github ssh key, which will be used to download dependencies in a secure manner, not persisting in any build layer
+
+### Deploying Name Bazaar smart contracts
+
+#### Deployment via ClojureScript deployer
+
+_Currently you need to be using Node 8 and a temporary downgrade of better-sqlite3 to `[better-sqlite3 "4.0.0"]` in `project.clj`!_
```

Comment:
Mention also why we need 8, so future devs (or us) know the reason.

Also, is this needed only for deployment?

Reply:
Right, I'll explain. And it's only for deployment.